package.xml

```diff
@@ -42,7 +42,4 @@
   <build_depend>message_runtime</build_depend>
   <run_depend>message_runtime</run_depend>
 
-  <build_depend>GTSAM</build_depend>
-  <run_depend>GTSAM</run_depend>
```

Comment:
This should no longer be removed as per https://github.com/streetdrone-home/LIO-SAM/pull/2

Reply:
This is because the commit with that was not merged into master but the CVAT submodule still pointed to that so this work had that in it as well. In the future we should only update submodules to point to commits that have landed in master in their repository.

Will now rebase to remove this commit :+1: